apps/aeutils/src/aeu_info.erl

```diff
@@ -37,3 +48,24 @@ read_trimmed_file(Filename) ->
 
 trim_ending_whitespace(Binary) ->
     re:replace(Binary, "\\s+$", "", [{return, binary}, global]).
+
+cached_file(Filename) ->
+    cached_file(Filename, fun read_trimmed_file/1).
+
+cached_file(Filename, ReadFun) ->
```

Comment:
Why not persistent_term?

Reply:
I think persistent_term should be used very sparingly, considering the update costs.
> Storing or updating a term (using put/2) is proportional to the number of already created persistent terms because the hash table holding the keys will be copied. In addition, the term itself will be copied.

http://erlang.org/doc/man/persistent_term.html